src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -302,7 +300,7 @@ public CatalogObjectMetadataList create(
                                                                                        commitTimeRaw);
             return ResponseEntity.ok(catalogObjectMetadata);
         } catch (RevisionNotFoundException e) {
-            return ResponseEntity.notFound().build();
+            return new ResponseEntity(HttpStatus.NOT_FOUND);
```

Comment:
better with the builder no?

Reply:
No it makes the swagger model generation fail

Reply:
same as upper response